ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -537,23 +549,23 @@ spec:
     supported: false
   relatedImages:
   - name: ako-operator
-    image: projects.registry.vmware.com/ako/ako-operator@sha256:cdebc910497c3eaf3624c84fd9e27600b13b53cf3142d098dff0742cb16c624b
+    image: projects.registry.vmware.com/ako/ako-operator@sha256:ad55b86975cd73dfd5b3ff1179c03a6cfe47f578c5c8d79ad37d5ce2aab6482b
   keywords:
   - AKO
   - AKO Operator
   - Load Balancer
   - Ingress Controller
   links:
   - name: AKO Operator
-    url: https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/release-1.8.2-operator/ako-operator#readme
+    url: https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/release-1.9.1-operator/ako-operator#readme
```

Comment:
should operator version be `1.9.3` all places?

Reply:
Have not made the bundle changes yet. Will add it in the next PR with updated `sha` for ako-operator:1.9.3 image once this PR is merged and image is generated.